.github/workflows/ci.yml

```
@@ -82,3 +82,24 @@ jobs:
         run: |
           pip install --user codecov
           /home/runner/.local/bin/codecov
+
+  test-on-s390x:
+    runs-on: ubuntu-22.04
+    steps:
+    - uses: actions/checkout@v3
+    - name: Setup emulator
+      run: |
+        sudo docker run --rm --privileged tonistiigi/binfmt:qemu-v6.2.0
+    - name: Run build
+      uses: uraimo/run-on-arch-action@v2.6.0
+      with:
+        arch: s390x
+        distro: ubuntu22.04
+        run: | 
+         apt-get update -y && DEBIAN_FRONTEND=noninteractive apt-get install -y rebar3 gcc libssl-dev
+         echo "---rebar3 as test get-deps---"
+         rebar3 as test get-deps
+         echo "---rebar3 as test compile---"
+         rebar3 as test compile
+         echo "---rebar3 as test ct---"
+         rebar3 as test ct
```

I'd actually separate the commands into different steps, this way GHA itself will already build the `echo` printouts and we get more granularity about what is it that failed and took too long
```suggestion
        run: apt-get update -y && DEBIAN_FRONTEND=noninteractive apt-get install -y rebar3 gcc libssl-dev
        run: rebar3 as test get-deps
        run: rebar3 as test compile
        run: rebar3 as test ct
```

---

I think this syntax is not supported with run-on-arch. Let me confirm.

---

@NelsonVides Above doesn't work, it throws "'run' is already defined" during startup. Also I checked [some examples](https://github.com/uraimo/run-on-arch-action/tree/master/.github/workflows), however could not find multiple run statements allowed.

---

Right, I see 🤔 what about the `install` step, maybe the `apt-get` command can be moved to a prior `install`, as in https://github.com/uraimo/run-on-arch-action/blob/master/.github/workflows/advanced-example.yml,
something like the following:
```suggestion
        install: |
          apt-get update -y
          DEBIAN_FRONTEND=noninteractive apt-get install -y rebar3 gcc libssl-dev
        run: | 
          echo "---rebar3 as test get-deps---"
          rebar3 as test get-deps
          echo "---rebar3 as test compile---"
          rebar3 as test compile
          echo "---rebar3 as test ct---"
          rebar3 as test ct
```

---

Thank you, added above change